3.7/ubuntu/Dockerfile

```diff
@@ -0,0 +1,138 @@
+# The official Canonical Ubuntu Bionic image is ideal from a security perspective,
+# especially for the enterprises that we, the RabbitMQ team, have to deal with
+FROM ubuntu:18.04 AS release
+
+FROM release AS openssl-otp
+
+ARG OTP_VERSION="21.2.2"
+# It would be nice if the Ericsson team published these checksums somewhere, will ask on the Erlang mailing list
```

Comment:
Only crickets so far... http://erlang.org/pipermail/erlang-questions/2019-January/097048.html

Reply:
That is a good find @lukebakken, thank you. Since only minor OTP releases are published, and everything else is expected to be applied as patches, the MD5 checksums are not really helpful. I've followed-up on the initial Erlang Questions thread: http://erlang.org/pipermail/erlang-questions/2019-January/097062.html

Reply:
FWIW, this is the Travis build that "deployed" OTP 21.2 to GitHub: https://travis-ci.org/erlang/otp/jobs/466456148